src/main/java/se/sundsvall/chat/api/model/Participant.java

```diff
@@ -0,0 +1,63 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.Objects;
+
+@Schema(description = "Represents a participant in a conversation, including their type and value.")
+public class Participant {
+	@Schema(description = "The type of the participant, e.g., 'user', 'bot', etc.", example = "user")
```

Comment:
I think that we should reference the supported types here (as defined in dept44:s Identifier):
adAccount
partyId